packages/farcaster-js/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { HubRestAPIClient, FarcasterEpochTimestamp } from '@standard-crypto/farcaster-js-hub-rest';
+export { NeynarAPIClient } from '@standard-crypto/farcaster-js-neynar';
```

Comment:
I think this needs to export ReactionType for a few of the endpoints to work. I could also refactor to take in a string

Reply:
https://github.com/standard-crypto/farcaster-js/blob/0e3341b62e873cda1e6f4ef9b5f8b7dcc155e680/packages/farcaster-js-neynar/src/v2/NeynarV2APIClient.ts#L318